src/components/login.js

```diff
@@ -0,0 +1,89 @@
+/* this is a code for login page
+Import combination
+import text field from material ui to get text box
+import link from material ui for buttons
+import formcontrllabel to display an extra label in checkbox
+import material ui checkbox to turn an option 
+import material ui grid to make responsive
+import css to get all properties defined
+import material ui typing component typography
+import material ui box all style functions can be used
+
+*/
+import React, { Component } from 'react';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Grid from '@material-ui/core/Grid';
+import '../App.css';
+import { FacebookLoginButton } from "react-social-login-buttons";
+import { GoogleLoginButton } from "react-social-login-buttons";
+class Login extends Component {
+  render() {
+
+    return (
+      <Grid item xs={12}>
+        <Grid item xs={12} sm={6} class="login_box">
+          <div className="main">
```

Comment:
Where are these classes? I think you have added classes to App.css or somewhere and forgot to check-in the changes.